sdcm/tester.py

```diff
@@ -2637,6 +2638,7 @@ def tearDown(self):
         self.log.info('TearDown is starting...')
         self.stop_timeout_thread()
         self.stop_event_analyzer()
+        self.stop_event_handler()
```

Comment:
shouldn't the event handler be stopped before the event analyzer?

Reply:
I'm not sure if this makes difference, but right, let's stop in the reversed order of starting